sop/modules/OSD/partials/ocs_extend_cluster.adoc

```diff
@@ -0,0 +1,27 @@
+
+To Extend the cluster threshold ratio ,  follow the below mentioned steps
+
+*Extend the Ceph cluster full threshold*
+
+Execute the following command to extend the ceph cluster full threshold
+[source,role="execute"]
+----
+oc process -n openshift-storage ocs-extend-cluster | oc create -f -
+----
+
+Now the ceph cluster threshold is extended from 0.85 to 0.87, this will make the cluster to get out of HEALTH_ERR state.
```

Comment:
```suggestion
Now the ceph cluster threshold is extended from 0.85 to 0.87, this will make the cluster to get out of HEALTH_ERR state and Readonly mode.
```